src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/RecallNotificationGotenbergComponentTest.kt

```diff
@@ -30,11 +32,12 @@ class RecallNotificationGotenbergComponentTest : GotenbergComponentTestBase() {
       currentPrisonId = PrisonId("MWI")
     )
 
-    val recallNotification = authenticatedClient.getRecallNotification(recall.recallId)
+    val recallNotificationId = authenticatedClient.generateDocument(recall.recallId, RECALL_NOTIFICATION)
+    val recallNotification = authenticatedClient.getDocument(recall.recallId, recallNotificationId.documentId)
 
 //    writeBase64EncodedStringToFile("recall-notification.pdf", recallNotification.content)
-    assertThat(recallNotification, hasNumberOfPages(equalTo(3)))
-    assertThat(recallNotification, hasTotalPageCount(3))
+    assertThat(Pdf(recallNotification.content), hasNumberOfPages(equalTo(3)))
```

Comment:
Are `hasNumberOfPages ` and `hasTotalPageCount ` actually duplicates?  
I think so.  In which case bin one in favour of the other of course.

Reply:
no, one checks the number of pages, and the other checks that the page count is correct. I'm pretty sure you wrote them, maybe it was Mike